libavformat/avio.c

```diff
@@ -407,13 +407,7 @@ int ffurl_read(URLContext *h, unsigned char *buf, int size)
 {
     if (!(h->flags & AVIO_FLAG_READ))
         return AVERROR(EIO);
-
```

Comment:
I believe we still this for non-multicast inputs, in the event that we're not using custom IO.

Reply:
Right! I think I merged this in when I moved to ssimwave-4-3. I'll pull this change out.